bower.json

```diff
@@ -17,6 +17,10 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-prelude": "^3.0.0"
+    "purescript-prelude": "^3.0.0",
+    "purescript-foldable-traversable": "^3.3.0"
```

Comment:
It also feels a bit odd to me. But then so does putting it in `purescript-foldable-traversable`. Curious to read your idea.